js/main.js

```diff
@@ -0,0 +1,261 @@
+// document.addEventListener('DOMContentLoaded', init)
+
+// var bs = burgerStack()
+
+// function init () {
+// userinput hamburger
+
+// setTimeout(function () {alert("DO READ INSTRUCTIONS BEFORE START")}, 1000)
+  var u1 = document.getElementById('u1')
+  var u2 = document.getElementById('u2')
+  var u3 = document.getElementById('u3')
+  var u4 = document.getElementById('u4')
+  var u5 = document.getElementById('u5')
+  var u6 = document.getElementById('u6')
+  var u7 = document.getElementById('u7')
+  var allUs = [u1, u2, u3, u4, u5, u6, u7]
+  var hamItem = document.querySelectorAll('.hamItem')
+// generated hamburger
+  var d1 = document.getElementById('d1')
+  var d2 = document.getElementById('d2')
+  var d3 = document.getElementById('d3')
+  var d4 = document.getElementById('d4')
+  var d5 = document.getElementById('d5')
+  var d6 = document.getElementById('d6')
+  var d7 = document.getElementById('d7')
+  var allDs = [d1, d2, d3, d4, d5, d6, d7]
+  var hamObj = document.querySelectorAll('.hamObj')
+  var gameStart = false
+  var counter = 0
+  var highScore = 0
+  var timer = 60
+  var timeCount = document.querySelector('.timer')
+  var inputShake = document.querySelector('.userinput')
+  var interval = null
+
+  document.addEventListener('keyup', onKeyUp)
+  var index = 4
+  // player chooses ingredients to the userinput screen
+  function onKeyUp (event) {
+    if (checkKeyCode(event.keyCode)) {
+      if (event.keyCode === 81) {
+        hamItem[index].classList.remove('hamItem')
+        hamItem[index].classList.add('cucumber')
+        index--
+        // if (checkIngredient() === false )
+      }
+      if (event.keyCode === 87) {
+        hamItem[index].classList.remove('hamItem')
+        hamItem[index].classList.add('meat')
+        index--
+      }
+      if (event.keyCode === 69) {
+        hamItem[index].classList.add('onion')
+        hamItem[index].classList.remove('hamItem')
+        index--
+      }
+      if (event.keyCode === 82) {
+        hamItem[index].classList.add('lettuce')
+        hamItem[index].classList.remove('hamItem')
+        index--
+      }
+      if (event.keyCode === 84) {
+        hamItem[index].classList.add('cheese')
+        hamItem[index].classList.remove('hamItem')
+        index--
+      }
+      if (event.keyCode === 32) {
+        u1.classList.add('topbun')
+        if (checkIngredient()) {
+          setTimeout(function () {
+            resetNow()
+            gamePlay()
+          }, 1000)
+          counter = counter + 10
+          point()
+        } else {
+          inputShake.classList.add('shake-hard', 'shake-constant')
```

Comment:
what's the different between `shake-hard` and `shake-constant` class? Explain in class and in`readme` please :)